build-ss-R-CMD-check: macOS-latest (release) error: "mv: rename ss_osx to ss: No such file or directory"

@MOshima-PIFSC 
"macOS-latest (release)" errors relate to filenname format issue: `ss_osx`

```
Run curl https://api.github.com/repos/nmfs-stock-synthesis/stock-synthesis/releases/latest | grep "browser_download_url" | grep -Eo 'https://[^\"]*' | grep "ss_osx" | xargs wget
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   277  100   277    0     0   3487      0 --:--:-- --:--:-- --:--:--  3644
mv: rename ss_osx to ss: No such file or directory
Error: Process completed with exit code 1.
```

The **macOS-latest workflow** from github action script:

https://github.com/PIFSCstockassessments/ss3diags/blob/d1d813c55e21a87a27a6e6c5884fed7aa5f5cb6c/.github/workflows/R-CMD-check.yaml#L48-L55

--- a/.github/workflows/R-CMD-check.yaml
+++ b/.github/workflows/R-CMD-check.yaml
@@ -45,14 +45,18 @@ jobs:
           mv ss_win.exe ss.exe
           cp ss.exe inst/extdata/ss.exe
 
-      - name: Get the latest SS3 executable for macOS and move to expected location
+      - uses: nick-fields/retry@v2
         if: matrix.config.os == 'macOS-latest' 
-        run:  |
-          curl https://api.github.com/repos/nmfs-stock-synthesis/stock-synthesis/releases/latest | grep "browser_download_url" | grep -Eo 'https://[^\"]*' | grep "ss_osx" | xargs wget
-          mv ss_osx ss
-          sudo chmod a+x ss
-          cp ss inst/extdata/ss
-          rm ss
+        with:
+          timeout_minutes: 3
+          max_attempts: 3
+          retry_on: error
+          command:  |
+            curl https://api.github.com/repos/nmfs-stock-synthesis/stock-synthesis/releases/latest | grep "browser_download_url" | grep -Eo 'https://[^\"]*' | grep "ss_osx" | xargs wget
+            mv ss_osx ss
+            sudo chmod a+x ss
+            cp ss inst/extdata/ss
+            rm ss
       
       - uses: r-lib/actions/setup-pandoc@v2
 

